test/clt-tests/core/test-skipping-plugin-loading.rec

```diff
@@ -0,0 +1,194 @@
+––– block: ../base/start-searchd –––
+––– input –––
+mysql -h0 -P9306 -e "SHOW BUDDY PLUGINS\G;" | grep "manticoresoftware/buddy-plugin-replace"
+––– output –––
+Package: manticoresoftware/buddy-plugin-replace
+––– input –––
+mysql -v -h0 -P9306 -e "SHOW TABLES; DROP TABLE IF EXISTS tbl; SHOW TABLES; CREATE TABLE tbl (id BIGINT, model TEXT, storage_capacity INTEGER, color string, release_year INTEGER, price FLOAT, discounted_price FLOAT, sold BOOL, date_added TIMESTAMP, product_codes MULTI, values MULTI64, additional_info JSON, vector float_vector knn_type='hnsw' knn_dims='4' hnsw_similarity='l2');"
+––– output –––
+--------------
+SHOW TABLES
+--------------
+--------------
+DROP TABLE IF EXISTS tbl
+--------------
+--------------
+SHOW TABLES
+--------------
+--------------
+CREATE TABLE tbl (id BIGINT, model TEXT, storage_capacity INTEGER, color string, release_year INTEGER, price FLOAT, discounted_price FLOAT, sold BOOL, date_added TIMESTAMP, product_codes MULTI, values MULTI64, additional_info JSON, vector float_vector knn_type='hnsw' knn_dims='4' hnsw_similarity='l2')
+--------------
+––– input –––
+chmod +x ./test/clt-tests/core/generate-1mln-records.sh
+––– output –––
+––– input –––
+total_iterations=20 ./test/clt-tests/core/generate-1mln-records.sh
```

Comment:
Is it okay to have 20 total iterations but a count of 1000? This might be confusing for those who do not know what the script does.

Reply:
Done